HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -466,6 +466,49 @@ public async Task<IActionResult> NotTakingPart_Post(IneligibleViewModel viewMode
         return RedirectToAction(forwardArgs.Action, forwardArgs.Controller, forwardArgs.Values);
     }
     
+    [HttpGet("not-participating")]
+    public IActionResult NotParticipating_Get(QuestionFlowStep? entryPoint, bool emailPreferenceSubmitted = false)
+    {
+        var questionnaire = questionnaireService.GetQuestionnaire();
+        var viewModel = new NotParticipatingViewModel
+        {
+            LocalAuthorityName = questionnaire.LocalAuthorityName,
+            Submitted = emailPreferenceSubmitted,
+            EmailAddress = questionnaire.NotificationEmailAddress,
+            LocalAuthorityMessagePartialViewPath = GetLocalAuthorityNotParticipatingMessagePartialViewPath(questionnaire),
+            CanContactByEmailAboutFutureSchemes = questionnaire.NotificationConsent.ToNullableYesOrNo(),
+            EntryPoint = entryPoint,
+            BackLink = GetBackUrl(QuestionFlowStep.NotTakingPart, questionnaire, entryPoint)
+        };
+
+        return View("NotParticipating", viewModel);
+    }
+    
+    [HttpPost("not-participating")]
+    public async Task<IActionResult> NotParticipating_Post(IneligibleViewModel viewModel)
+    {
+        if (!ModelState.IsValid)
+        {
+            return NotParticipating_Get(viewModel.EntryPoint, false);

```

Comment:
You don't need to provide `false` as a parameter here, since it is defaulted to `false` above.